src/backend/tests/work-packages.test.ts

```diff
@@ -0,0 +1,54 @@
+import request from 'supertest';
```

Comment:
there is already a tests folder and a work-packages.test.ts file one directory above this

Reply:
Where would I find the other tests folder? I can only seem to find one on my end

Reply:
one directory above this. backend/tests